python/ray/actor.py

```diff
@@ -590,6 +593,8 @@ def _remote(self,
                 this actor or task and its children (see
                 :ref:`runtime-environments` for details).  This API is in beta
                 and may change before becoming stable.
+            execute_out_of_order (bool): Allow the actor to execute tasks out
```

Comment:
Should this be exposed in the top-level API? How about enabling it by default for async/concurrent actors and keeping it hidden?

Reply:
+1 for this!